rgw/v2/tests/s3_swift/configs/test_bucket_notification_kafka_broker_multipart_with_kafka_acl_config_set.yaml

```diff
@@ -0,0 +1,22 @@
+config:
```

Comment:
Please include test script name and Polarion ID here

Reply:
done @ckulal 
please re-review